pkg/policytools/policytools.go

```diff
@@ -14,11 +14,11 @@ type Cmd struct{}
 func (a Cmd) GetCmd() *cobra.Command {
 	policyCmd := &cobra.Command{
 		Use:   "policytools",
-		Short: "ACM Policy Toolset",
-		Long: `ACM Policy Toolset
+		Short: "RHACM Policy Toolset",
```

Comment:
RHACM isn't an approved short name

Reply:
Okay--I'll revert the change back to ACM.

Reply:
ACM isn't approved either lol it should be Red Hat Advanced Cluster Management. But I saw you used it for the long version